frontend/src/static/js/v_summary.js

```diff
@@ -505,6 +507,13 @@ window.vSummary = {
         merged[key] += value;
       });
     },
+    getRandomColor() {
+      let color = '#';
+      while (color.length < 7) {
+        color += hexLetters[Math.round(Math.random() * 15)];
+      }
+      return color;
+    },
```

Comment:
can use this one line code to generate random color since ffffff == 16777215
return '#' + Math.round(Math.random()*16777215).toString(16)